src/adaptor/command.ts

```diff
@@ -0,0 +1,32 @@
+import type { Message, PartialMessage } from 'discord.js';
+import type { Lifter } from '.';
+import type { Snowflake } from '../model/id';
+import type { CommandMessage } from '../service/command-message';
+import { convertEmbed } from './embed-convert';
+
+const SPACES = /\s/;
+
+export const converterWithPrefix = (
+  prefix: string
+): Lifter<CommandMessage> => ({
+  lift:
+    (func: (command: CommandMessage) => Promise<void>) =>
+    async (message: Message | PartialMessage): Promise<void> => {
+      if (!message.content?.trimStart().startsWith(prefix)) {
+        return;
+      }
+      const args = message.content
+        ?.trim()
+        .slice(prefix.length)
+        .split(SPACES)
+        .filter((arg) => arg !== '');
```

Comment:
`SPACES = /\s+/`にすれば`filter()`は要らないかと
```suggestion
        .split(SPACES)
```

Reply:
9e946de で修正しました.